index.html

```diff
@@ -22,5 +24,44 @@
       </div>
       <a href="">Download</a>
     </nav>
+    <section class="askedQuestion">
```

Comment:
small change, for naming convention, perhaps better to use `asked-questions` for the class name